src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -132,6 +132,7 @@ trait mappedPropVals {
         f.get(this) match {
           case x: String => f.getName -> x
           case y: Option[_] => f.getName -> y.asInstanceOf[Option[_]].getOrElse("").toString
+          case z => f.getName -> z.toString
```

Comment:
Maybe the explicitness is nice, but if you put this at the top you would eliminate the need for the String case.